images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -98,16 +98,29 @@ RUN ${IMAGES_DIR}/retry.sh "curl -o /usr/local/share/ca-certificates/verisign.cr
     
 # Install PHP pre-reqs	# Install PHP pre-reqs
 RUN if [ "${DEBIAN_FLAVOR}" = "buster" ]; then \
-    apt-get update \
-    && apt-get upgrade -y \
-    && apt-get install -y \
-        $PHPIZE_DEPS \
-        ca-certificates \
-        curl \
-        xz-utils \
-        libsodium-dev \
-        libncurses5 \
-    --no-install-recommends && rm -r /var/lib/apt/lists/* ; \
+        apt-get update \
+        && apt-get upgrade -y \
+        && apt-get install -y \
+            $PHPIZE_DEPS \
+            ca-certificates \
+            curl \
+            xz-utils \
+            libsodium-dev \
+            libncurses5 \
+        --no-install-recommends && rm -r /var/lib/apt/lists/* \
+        # A workaround for php 8.0 composer issue: https://github.com/microsoft/Oryx/issues/1100
```

Comment:
Could you please provide the source of the workaround as well? I imagine they provide more explanation there, so it would be easier to determine what all the commands do.

Reply:
[software installation - How to have both libcurl3 and libcurl4 installed at same time? - Ask Ubuntu](https://askubuntu.com/questions/1061431/how-to-have-both-libcurl3-and-libcurl4-installed-at-same-time)

[Using libcurl3 and libcurl4 on Ubuntu 18.04 (Bionic) - DEV Community](https://dev.to/jake/using-libcurl3-and-libcurl4-on-ubuntu-1804-bionic-184g)

Reply:
Trying to summarize what the commands did here: add symlink for libonig library; remove existing libcurl3 and curl tool; install libcurl4 from source; reinstall curl tool. Please let me know if questions coming up, thanks!